evidence/alexggh/0002-2025-01-20-rank-1-retention.md

```diff
@@ -0,0 +1,53 @@
+# Evidence-0001: Retention at Rank I
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | 2025/01/20                                                                                  |
+| **Submitted by**| Alexandru Gheorghe                                                                          |
+
+
+## Member details
+
+- Matrix username: @alexggh:parity.io
+- Polkadot address: 14uA7Vc828e2Q6oL5GBHP9UzTkEvwqbroERwRmucGrLmPuuL
+- Current rank: 1
+- Date of initial induction: 2024/07/20
+- Date of last report: 2024/10/21
+- Area(s) of Expertise/Interest: Parachains consensus
+
+
+## Reporting period
+
+- Start date: 2024/10/21
+- End date: 2025/01/20
+
+
+## Evidence
+During the reporting I was mostly focused on the scalability and stability of the Polkadot network.
+
+On the scalability side, the end goal is expanding the number of validators and cores to 1000 validators and 200 cores. To achieve that I've been directly involved in the following work streams:
+
+- Optimize and parallelize approval-voting subsystems. Approval voting protocol is one of the system that scales linearly with the number of validators and cores, on top of that it needs to be  fast in processing its messages, because every 500ms validators trigger new work if they did not gathered enough assignments from their peers. To that end a major rearchitecting of these subsytems was done to make sure we parallelize the processing and optimize the hot paths, the implementation was delivered with [4849](https://github.com/paritytech/polkadot-sdk/pull/4849), [4848](https://github.com/paritytech/polkadot-sdk/pull/4848) [4928](https://github.com/paritytech/polkadot-sdk/pull/4928), [5411](https://github.com/paritytech/polkadot-sdk/pull/5411).
```

Comment:
```suggestion
- Optimize and parallelize approval-voting subsystems. Approval voting protocol is one of the systems that scale linearly with the number of validators and cores, on top of that it needs to be fast in processing its messages, because every 500ms validators trigger new work if they did not gathered enough assignments from their peers. To that end a major rearchitecting of these subsystems was done to make sure we parallelize the processing and optimize the hot paths, the implementation was delivered with [4849](https://github.com/paritytech/polkadot-sdk/pull/4849), [4848](https://github.com/paritytech/polkadot-sdk/pull/4848) [4928](https://github.com/paritytech/polkadot-sdk/pull/4928), [5411](https://github.com/paritytech/polkadot-sdk/pull/5411).
```